<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDashboard</class>
 <widget class="QWidget" name="StudentDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#sidebarFrame {
 border-radius: 10px;
 border: 1px solid #000000;
}

#topbarFrame {
	background-color: none;
	border: none;
}

#dashboardButton {
    background-color: #ffffff;
	border-radius: 10px;
	color: black;
	height: 32px;
	width: 120
}
#subjectsButton {
    background-color: #ffffff;
	border-radius: 10px;
	color: black;
	height: 32px;
	width: 120
}
#certificatesButton {
    background-color: #ffffff;
	border-radius: 10px;
	color: black;
	height: 32px;
	width: 120
}

#dashboardLogoutButton {
	background-color: rgb(59, 17, 36);
	border-radius: 10px;
	color: white;
	height: 32px;
	width: 120
}

#searchLineEdit {
    border-radius: 10px;
    background-color: rgb(255, 255, 255);
    color: #000000;
    height: 32px;
	padding-left: 10px;
}

#addButton {
	background-color: #ffffff;
	border-radius: 10px;
	color: black;
     font-weight: bold;
}

#contentFrame {
 border-radius: 10px;
 border: 1px solid #000000;
}

#appTitle {
 font-weight: bold;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <widget class="QFrame" name="sidebarFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="sidebarLayout">
      <item>
       <widget class="QLabel" name="appTitle">
        <property name="text">
         <string>STUDYSYNC</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dashboardButton">
        <property name="text">
         <string>Dashboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="subjectsButton">
        <property name="text">
         <string>Subjects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="productivityButton">
        <property name="styleSheet">
         <string notr="true">background-color: #ffffff;
border-radius: 10px;
color: black;
height: 32px;
width: 120px;
</string>
        </property>
        <property name="text">
         <string>Productivity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="certificatesButton">
        <property name="text">
         <string>Certificates</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="sidebarSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="dashboardLogoutButton">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="contentFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="topMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="homeStackedWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="dashboardPageStack">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="dashboardPageLabel">
            <property name="text">
             <string>DASHBOARD PAGE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="subjectsPageStack">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="subjectsPageLabel">
            <property name="text">
             <string>SUBJECTS PAGE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="certsPageStack">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="certPageLabel">
            <property name="text">
             <string>CERTIFICATES PAGE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="prodPageStack">
         <property name="styleSheet">
          <string notr="true">QWidget {
    font-family: 'Segoe UI', 'Arial', sans-serif;
    font-size: 14px;
    color: #333;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    padding: 8px 16px;
    font-size: 14px;  /* Ensures same font size */
    font-family: 'Segoe UI', 'Arial', sans-serif;  /* Ensures same font */
    border: none;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #45a049;
}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(17, 25, 34);
color: white;

</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QScrollArea" name="scrollArea">
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>738</width>
                  <height>800</height>
                 </rect>
                </property>
                <widget class="QLabel" name="prodTrack">
                 <property name="geometry">
                  <rect>
                   <x>160</x>
                   <y>10</y>
                   <width>381</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#prodTrack
 {
    qproperty-alignment: 'AlignCenter';
    font-size: 18px;
    font-weight: bold;
}
</string>
                 </property>
                 <property name="text">
                  <string>Productivity Tracker</string>
                 </property>
                </widget>
                <widget class="QLabel" name="date">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>50</y>
                   <width>100</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Date</string>
                 </property>
                </widget>
                <widget class="QDateEdit" name="dateEdit">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>80</y>
                   <width>204</width>
                   <height>54</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="subject">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>140</y>
                   <width>100</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Subject</string>
                 </property>
                </widget>
                <widget class="QComboBox" name="subjectPick">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>160</y>
                   <width>204</width>
                   <height>54</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="studyDuration">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>220</y>
                   <width>131</width>
                   <height>31</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Duration (min)</string>
                 </property>
                </widget>
                <widget class="QSpinBox" name="timeSpent">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>260</y>
                   <width>71</width>
                   <height>54</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="notes">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>330</y>
                   <width>100</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">selection-color: rgb(229, 47, 255);
color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>Notes</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="noteEdit">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>350</y>
                   <width>204</width>
                   <height>86</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QPushButton" name="addSesh">
                 <property name="geometry">
                  <rect>
                   <x>300</x>
                   <y>430</y>
                   <width>131</width>
                   <height>32</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Add Session</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="customSubject">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>160</y>
                   <width>181</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QPushButton" name="addSubButton">
                 <property name="geometry">
                  <rect>
                   <x>290</x>
                   <y>210</y>
                   <width>61</width>
                   <height>32</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
                <zorder>customSubject</zorder>
                <zorder>prodTrack</zorder>
                <zorder>date</zorder>
                <zorder>dateEdit</zorder>
                <zorder>subject</zorder>
                <zorder>subjectPick</zorder>
                <zorder>studyDuration</zorder>
                <zorder>timeSpent</zorder>
                <zorder>notes</zorder>
                <zorder>noteEdit</zorder>
                <zorder>addSesh</zorder>
                <zorder>addSubButton</zorder>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
